packages/api-explorer/src/Doc.jsx

```diff
@@ -393,10 +393,12 @@ Doc.propTypes = {
   language: PropTypes.string.isRequired,
   lazy: PropTypes.bool,
   Logs: PropTypes.func,
+  maskErrorMessages: PropTypes.bool,
```

Comment:
How does this property get sent down from ReadMe?

Reply:
Haven't done that part yet, but the idea was to expose the `isAdmin` boolean from hub2 users to a `<script data-json="..." />` tag  our API Reference template in the same way we populate the rest of the Explorer.